src/datepicker/datepicker.js

```diff
@@ -150,7 +151,9 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.position'])
       }
 
       // Configuration parameters
-      var mode = 0, selected = new Date(), showWeeks = datepickerConfig.showWeeks;
+      var mode = 0;
+      var selected = scope.focusedDate = new Date();
```

Comment:
Ok. I don't think you need it in the `scope` definition to be able to watch it. Simply setting `scope.focusedDate` like you do here will allow that, because anything on the `scope` object can be watched.